### ISSUE TITLE
Questions from Catch Game internal playtest

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-gui#409]
**TL;DR:** Along the same general lines as https://github.com/LLK/scratch-gui/issues/233, these are questions that came up during the Catch Game internal playtest (6/2/17) that seem worth consideration but awkward as individual issues. (Discussed how best to organize them with @carljbowman. 👍 )

Reported by @ntlrsk:
> I had expected "touching <mouse pointer> to show the latest (other) sprite name, but realize now that's not how it works in Scratch 2.0. But something to consider. mouse pointer is more consistent, but much rarer than wanting to sense another sprite 

Group question:
> When should things be in the dropdown menus vs. separate blocks?

Relatedly, we were noticing that options from the dropdown menu may get used very rarely; for example, a lot of people were surprised to learn about go to [random position] but were familiar with go to [mouse pointer].

@fsih reported:
> Random idea: what if you could click on a shadow block and get a dropdown of suggestions for what you could put in there? In particular it takes a really long time to search around the block categories and assemble enough blocks to write "if x > 10 and x < threshold" but what if you could click on the diamond in the 'if' block and it suggested all those operators to you in a dropdown

@jwzimmer reported: "Go to x vs. set x to _ ?"
I was just wondering about what the difference in "go to" and "set" in the blocks meant - it looks like maybe "go to" involves a pair of coordinates (x,y) whereas "set" refers to a single value? I was wondering if the distinction is important or if we need both "go to" and "set" since if you "set" x the sprite also "goes to" that x value (and vice versa)?

@jwzimmer reported: "Should "go to _" block accept "pick random"?"
Another question about whether it's fine for things to fit into blocks they don't make a lot of sense with (I think it probably is?).

@jwzimmer reported: "Width of green flag hat block being so small compared to other blocks makes it hard to read"
If you compare the width of the hat block in 3.0 versus in the 2.0 cards, the hat block does not stick out as far at the top of the stack (in 3.0). I think this makes it a little harder to read/ identify each stack.

@fsih reported:
> UI thing, I don't understand why it says "2" under my backdrop, I don't see 2 and clicking it doesn't show 2 things

@ntlrsk reported:
> When a forever script is running the stop sign is grayed out which makes it seem like it's not available to be clicked. Same with green flag before program is run, looks grayed out 

@lifeinchords reported confusion around Play sound vs. play sound until done.

Jie reported:
> If I don't have a Stop All block and run the script, and then add the Stop All while the script is running, code will not stop even after you reach greater than the win score.  May have something to do with not resetting properly (since I noticed the "win" sprite is visible even before actually winning)

> Script running without stop block
<img width="566" alt="screen shot 2017-06-05 at 10 47 40 am" src="https://cloud.githubusercontent.com/assets/1320456/26788826/7295f3ae-49dc-11e7-85e9-cd67c181e99f.png">

> Script still running after Stop All block added and winning score reached
<img width="443" alt="screen shot 2017-06-05 at 10 48 34 am" src="https://cloud.githubusercontent.com/assets/1320456/26788870/9832ada0-49dc-11e7-8325-e8548467e5b4.png">

Shruti reported confusion around when something was a previous stamp vs. the sprite itself when visible on stage:
> I deleted all the sprites, but still showing on the stage (it happens when I use stamp).

Jie reported:
> It's possible to put boolean inside number of repeats, but not sure what that's supposed to do?  Currently the block clicks in but nothing happens
<img width="590" alt="screen shot 2017-06-05 at 10 51 22 am" src="https://cloud.githubusercontent.com/assets/1320456/26789006/f2db60d0-49dc-11e7-9b54-651bd60009e6.png">

Group question:
> What are the ranges for the various things e.g. Pitch? It's hard to know what values to put into e.g. pick random when you don't know the range.

**Comment by jwzimmer** [comment]
From Catch Game internal playtest II (6/12/17):

@mewtaylor:
> Should the default "switch costume" selection be the next costume in the series, rather than the already present, default costume?

Group question:
> Should every sprite have at least 1 sound?
> (notably, the banana doesn't have any sounds, even though the cards ask you to use a banana sound)

**Comment by jwzimmer** [comment]
I will close this when the new issues we decided on are opened (working on that now).

**Comment by jwzimmer** [comment]
Closing, replaced by:
- https://github.com/LLK/scratch-gui/issues/446
- https://github.com/LLK/scratch-gui/issues/447
- https://github.com/LLK/scratch-gui/issues/448
- https://github.com/LLK/scratch-blocks/issues/943
- https://github.com/LLK/scratch-gui/issues/449
- https://github.com/LLK/scratch-gui/issues/450
- https://github.com/LLK/scratch-gui/issues/451
